libs/http/cors.go

```diff
@@ -0,0 +1,39 @@
+package http
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/rs/cors"
+)
+
+// CORSConfig represents the configuration for CORS.
+type CORSConfig struct {
+	AllowedOrigins string `long:"allowed-origins" description:"Allowed origins for CORS"`
```

Comment:
Could/Should that be a slice?

Reply:
@jeremyletang Could. Not sure if it should though. I mean, it was easier to put `string` because of `*`. I can change it if you think it'd be cleaner

Reply:
I don't mind, it's just that then the lib can setup a list of allowed origins, here we restrict it to 1. not that it matters much

Reply:
We can still have more than one, since the origin is matched with `strings.Contains` (line 37 below).
Yeah, now I realise it's not that clean. I better fix it